float32 bicep_error
float32 tricep_error
float32 bicep_ref
float32 tricep_ref
int8 cal_msg
int8 sat_bicep
int8 sat_tricep
